packages/cascara/src/ui/Form/Form.js

```diff
@@ -189,7 +189,7 @@ const Form = ({
     WARNING_STRINGS.INVALID_EDITING_AND_DISPLAY
   );
 
-  const { isEmpty } = getStatusFromDataLength(Object.keys(dataDisplay).length);
+  const { isEmpty } = getStatusFromDataLength(Object.keys(dataDisplay)?.length);
```

Comment:
Make sure we are checking for this value before using it. Our helper function should do the rest.